docs/articles/guides/console-args.md

```diff
@@ -43,12 +43,57 @@ You can also filter the benchmarks by categories:
 
 ## Runtimes
 
-The `--runtimes` or just `-r` allows you to run the benchmarks for selected Runtimes. Available options are: Clr, Mono, Core and CoreRT.
+The `--runtimes` or just `-r` allows you to run the benchmarks for selected Runtimes. Available options are: Mono, CoreRT, net46, net461, net462, net47, net471, net472, netcoreapp2.0, netcoreapp2.1, netcoreapp2.2, netcoreapp3.0.
```

Comment:
Do we really need `net46`, `net461`, `net47`, `net471`, `net472`?
When we are talking about a runtime for a benchmark, we typically want to run it on .NET Framework. Since it's impossible to install several versions of the .NET Framework side-by-side, we want to run it on the currently installed version of .NET Framework. However, we don't have this option anymore: we have to specify a particular version of the .NET Framework each time.

Reply:
@AndreyAkinshin I think that we can have both, I added `core` and `clr` back in https://github.com/dotnet/BenchmarkDotNet/commit/e7d4d96113e822393383afde8a2b83208f9861c0

PR coming on Monday ;)

Reply:
Awesome!